sharding-core/sharding-core-execute/src/main/java/org/apache/shardingsphere/core/execute/metadata/TableMetaDataLoader.java

```diff
@@ -165,8 +174,9 @@ private boolean isTableExist(final Connection connection, final String catalog,
                 String columnName = resultSet.getString(COLUMN_NAME);
                 String columnType = resultSet.getString(TYPE_NAME);
                 boolean isPrimaryKey = primaryKeys.contains(columnName);
-                boolean isNotNull = isPrimaryKey || !resultSet.getBoolean(IS_NULLABLE);
```

Comment:
Can `` not work well? I did a following test, and it returns true.
```
ResultSet resultSet = statement.executeQuery("select 'YES' ");
        resultSet.next();
        System.out.println(resultSet.getBoolean(1));
```
If resultSet.getString(IS_NULLABLE) == 'YES', resultSet.getBoolean(IS_NULLABLE) will be true. Is it right?

Reply:
I connect database to test resultSet.getBoolean(IS_NULLABLE) is not work.

![image](https://user-images.githubusercontent.com/10703753/68665099-67931f00-057c-11ea-9026-5200414b682b.png)
as image,only return "YES" OR "NO",if  getBoolean will throw exception

Reply:
MySQL Connector/J 8.0 + not work